doc/lsst.ap.pipe/pipeline-tutorial.rst

```diff
@@ -115,151 +79,38 @@ The result from running ``ap_pipe`` should look something like
    apdb/
       association.db   <--- the Alert Production Database with DIAObjects
    repo/
-      rerun/
-         processed/
-            repositoryCfg.yaml
-            deepDiff/
-               v123456/   <--- difference images and DIASource tables are in here
-            123456/   <--- all other processed data products are in here (calexps etc.)
-
-This is one example, and your rerun or output directory structure may differ.
-Of course, to inspect this data with the Butler, you don't need to know
-where it lives on disk. You should instead instantiate a Butler within python
-in the ``processed`` directory and access the data products that way.
+      contains_no_user_servicable_files/
```

Comment:
Ha! Is this really how it works? I couldn't open a calexp or a diffim output by ap_pipe with, e.g., ds9 if I wanted to?

Reply:
They're still regular files (for now?), but their organization is more complicated than in Gen 2 and my impression is that Gen 3 is less tolerant of user hacking than Gen 2 was. So I'm disinclined to create the kind of detailed directory tree you had in the original guide (also, as we saw with the collection names, it's very subject to change ATM).